starter-code/src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.strength = strength;
+    this.health = health;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  attack() {
```

Comment:
This method is useless because you wrote "class Viking extends Soldier". This means that your Viking class will inherit the attack() method (and all other methods) from the Soldier class.  ==> remove line 21-23